### DISCUSSION TITLE
Error in putty connection

### DISCUSSION THREAD

**Discussion by Maheshdvp** [learndevopsonline/learndevopsonline#1642] [Q&A] [answered]
Hi team, I just logged in to my aws account after few days and started my workstation instance and connected putty as well but after some time connection to putty was time outed and cant able connect I tried multiple times by stopping and starting that instance but connection was not established could  you please help me on that.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/116950128/4c9d1f12-5b1a-4d25-b797-e54b8739b0bb)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/116950128/2cd18b48-f882-45e7-960e-4d458b85fba3)

**Comment by sreedharm07** [comment, marked as answer]
check internet gateway attached to vpc or not.
and check security groups too

**Reply by Maheshdvp** [reply]
Internet gate way is attached and security group also allow all
![image](https://github.com/learndevopsonline/learndevopsonline/assets/116950128/8d711daa-a89f-44e2-b823-2fcde1ee07bd)

**Reply by sreedharm07** [reply]
from your screen shot i seen you have few instances, only one instance is not able to log in or every instance is getting same error?

**Comment by Maheshdvp** [comment]
Every instance same error

**Reply by sreedharm07** [reply]
Please join lab session 
Because i cont log in into your aws account to check the issue

**Comment by Maheshdvp** [comment]
Ok thank u
